yts/yts.py

```diff
@@ -124,146 +136,100 @@ def extract_movie_data(self, movie_data) -> dict:
         )
         return {movie_title: rating}
 
+
     # Execute Transmission-gtk with the downloaded torrent
-    def execute_transmission(self, torrent_name):
-        command = f"transmission-gtk {torrent_name}"
-        command_list = command.split()
-        subprocess.Popen(command_list)
+    @staticmethod
+    def execute_transmission(torrent_name):
+        subprocess.Popen(["transmission-gtk", torrent_name])
 
     # Utitility method to print available formats
     # Param: return of extract_format method
-    def print_formats(self, formats):
+    @staticmethod
+    def print_formats(formats=None, name=None, year=None):
+        if formats is None:
+            formats = yts.get_movie_data(name, year)[1]
+
         print("Available In:")
         for key in formats.keys():
             print("\t", key)
 
-
 # TODO design a better cli usage
 # Handle CLI arguments Class
 class Arguments:
-
     # Initialize arguments
     def __init__(self):
         self.arguments = self.get_cli_args()
-        self.check_one_flag_usage()
 
     # Handles CLI arguments
     # Returns a dictionary of the arguments
     # Only one flag is to be applied at a time
-    def get_cli_args(self) -> dict:
+    @staticmethod
+    def get_cli_args() -> dict:
         parser = argparse.ArgumentParser(prog="yts",
                                          description="Downloads YTS movie torrents.",
                                          allow_abbrev=False)
 
+        flags = parser.add_mutually_exclusive_group("Select one option for downloading.", required=True)
```

Comment:
```suggestion
        flags = parser.add_mutually_exclusive_group(required=True)
```
It takes only a single argument.